SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -1,72 +1,90 @@
 
- ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
+To enable the GDK toolbar, navigate to **Edit** > **Plugins** inside the Unreal Editor and scroll down to the bottom. Select the **SpatialOS Unreal GDK** section and enable the toolbar:
 
-* **Configuration**
+![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/enable-toolbar.png)
+
+#### Buttons
+
+The GDK toolbar has five features mapped to individual buttons and is displayed in the main editor toolbar to the right of the `Launch` button:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-buttons.png)
 
-    * **Project root folder**</br>
-    If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`.
+You can also access these from the **Window** menu:
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/window-access.png)
 
-    * **Launch configuration**</br>
-    The [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config) (SpatialOS documentation) to use when running `spatial local launch` using the `Launch` button.
+| Button | Description |
+| --- | --- |
+| Snapshot | Generates a [SpatialOS snapshot (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#snapshot). | 
+| Schema | Creates `schema` for your Unreal project. |
+| Launch | Runs [`spatial worker build build-config` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-worker-build-build-config) to build worker configs and runs `spatial local launch` with the launch configuration specified in the settings (see [below](#settings)). |
+| Stop | Stops `spatial local launch`. |
+| Inspector | Opens the [Inspector (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#inspector) in a browser. |
 
-    * **Stop on exit**</br>
-    If enabled, shuts down running deployments when you close the Unreal Editor.
+#### Settings
 
-    * **Snapshot path**</br>
-    Use this to specify the filepath of your snapshot. If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`.
+The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Toolbar**.
+
+ ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)
 
-    * **Snapshot file name**</br>
-    Name of your snapshot file.
+##### Configuration
 
-* **Schema Generation**
+| Setting | Description |
+| --- | --- |
+| SpatialOS directory | If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`. |
+| Launch configuration | The [launch configuration file (SpatialOS documentation)]({{urlRoot}}/content/glossary#launch-configuration) to use when running `spatial local launch` using the **Launch** button. |
```

Comment:
Another one I missed - remove `(SpatialOS documentation)` and change the link to `({{urlRoot}}/content/glossary#launch-configuration-file)`